cob_people_detection/ros/src/face_recognizer_node.cpp

```diff
@@ -518,7 +518,7 @@ void FaceRecognizerNode::facePositionsCallback(const cob_perception_msgs::ColorD
 				// set label
 				det.label = identification_labels[head][face];
 				// set origin of detection
-				det.detector = "face";
+				det.detector = ros::this_node::getName();
```

Comment:
I doubt that the detection tracker will work with these names without further modification. I am pretty sure that the tracking distinguishes detections by their type between head and face for updating confidences. Hence, add a fix for the detection tracker as well.

E.g. in https://github.com/ipa320/cob_people_perception/blob/indigo_dev/cob_people_detection/ros/src/detection_tracker_node.cpp#L450